Формула/Формула/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace Function
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Значение x:");
+
+            int x = int.Parse(Console.ReadLine());
+            int x1 = (int)(Math.Sqrt(x + 1) + Math.Sqrt(x - 1));
+            int x2 = (int)(Math.Sqrt(x));
+            int y = x1 / x2;
```

Comment:
Неверный формат данных.

> Составить программу, которая запрашивает у пользователя **действительное** значение _x_...

Конвертация в `int` в промежуточных вычислениях отбрасывает дробную часть, что приводит далее к неверному значению функции. Проще все вычисления написать в одном выражении, тогда будут не нужны дополнительные локальные переменные

```suggestion
            double x = double.Parse(Console.ReadLine());
            double y = (Math.Sqrt(x + 1) + Math.Sqrt(x - 1)) / Math.Sqrt(x);
```